src/uu/chroot/src/chroot.rs

```diff
@@ -234,13 +235,13 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 }
 
 pub fn uu_app() -> Command {
-    Command::new(uucore::util_name())
+    let cmd = Command::new(uucore::util_name())
         .version(uucore::crate_version!())
-        .help_template(uucore::localized_help_template(uucore::util_name()))
         .about(translate!("chroot-about"))
         .override_usage(format_usage(&translate!("chroot-usage")))
         .infer_long_args(true)
-        .trailing_var_arg(true)
+        .trailing_var_arg(true);
+    uucore::clap_localization::configure_localized_command(cmd)
```

Comment:
What's the reason for splitting the setup of command into two parts? Why not doing the localization at the end?